examples/stable/test/end_to_end/candid_rpc/workspaces/workspaces/siblings/src/index.ts

```diff
@@ -0,0 +1,11 @@
+import { IDL, query } from 'azle';
+
+export default class {
+    @query([], IDL.Text)
+    getAzleVersionFromSibling(): string {
```

Comment:
Make sure it's either `sibling` or `siblings`, the directory, the name, the suffix, etc

Reply:
siblings everywhere

Reply:
done